TsRandomizer/Screens/Settings/GameSettingsCollection.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+using TsRandomizer.Extensions;
+using TsRandomizer.Screens.Settings.GameSettingObjects;
+
+namespace TsRandomizer.Screens.Settings
+{
+	public class GameSettingsCollection
+	{
+		[JsonIgnore()]
+		public string Path { get; private set; }
+		public OnOffGameSetting DamageRando { get; set; }
+		public SpecificValuesGameSetting ShopFill { get; set; }
+		public NumberGameSetting ShopMultiplier { get; set; }
+		public OnOffGameSetting ShopWarpShards { get; set; }
+
+		public GameSettingsCollection()
+		{
+			string dir = "settings/";
+			if (!Directory.Exists(dir)) Directory.CreateDirectory(dir);
```

Comment:
plz put directory & file manipulation its its own sub method with a try/catch around them as a lot of things can go wrong when attempting to access the harddrive